### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#119] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by apirakas** [comment]
Hello @ludelafo @hadrylouis ,
you mentioned alternatives to https://desec.io/ yesterday. Can you share them with us ?

**Reply by ludelafo** [reply]
Yes, of course!

Here is a list of alternatives:

- http://www.duckdns.org/
- https://www.noip.com/
- https://freedns.afraid.org/

These alternatives are not as polished as deSEC but they are reliable and can be trusted.

I do not have any experience with these solutions but the steps are the same as described in the course material:

1. Create a new domain name
2. Add the `A` records to point to the external IP of your virtual machine
3. Wait a few minutes for the DNS propagation to complete (between 10-15 minutes from my experience)
4. Validate you can ping your virtual machine using the domain name
5. Validate the DNS records are fine using the `nslookup`/`dig` commands as mentioned in the course
6. Follow the steps regarding Traefik and you should be good to go.

Please let us know if you were able to complete these steps!

**Reply by apirakas** [reply]
perfect, I was able to create a domain and do the DNS registration with the https://www.noip.com/ site.

**Reply by ludelafo** [reply]
Great! Thanks for the update.
